AdyenNetworking/APIClient/APIClient.swift

```diff
@@ -98,7 +100,9 @@ public final class APIClient: APIClientProtocol {
     private func handle<R: Request>(_ result: Result<URLSessionSuccess, Error>,
                                     _ request: R,
                                     completionHandler: @escaping CompletionHandler<R.ResponseType>) {
-        requestCounter -= 1
+        DispatchQueue.main.async {
+            self.requestCounter -= 1
+        }
```

Comment:
maybe extract this into a function, maybe `incrementRequestCounter()`?

Reply:
it can be `incrementCounter(increment: Int)`
but i get your point :slightly_smiling_face: